product/views.py

```diff
@@ -67,6 +72,49 @@ def get(self, request):
 
         return JsonResponse({'products' : products_list, 'count' : products_count}, status=200)
 
+class ProductCartView(View):
+    @login_decorator
+    def post(self, request):
+        try:
+            user = request.user
+
+            if not Product.objects.filter(id=product_id).exists():
+                return JsonResponse({'message':'INVALID_PRODUCT'}, status=404)
+            
+            data     = json.loads(request.body)
+            color    = ProductColor.objects.get(name=data['color'])
+            size     = ProductSize.objects.get(name=data['size'])
+            quantity = int(data['quantity'])
+            product  = Product.objects.get(id=data['id'])
+            
+            if not ProductOption.objects.filter(
+                product=Product.objects.get(id=product_id),color=color, size=size
+            ).exists():
+                return JsonResponse({'message':'INVALID_PRODUCT_OPTION'}, status=404)
+
+            product_option = ProductOption.objects.get(
+                product=Product.objects.get(id=product_id),color=color, size=size
+            )
+            order = Order.objects.create(user=user, status=OrderStatus.objects.get(id=1))
```

Comment:
```suggestion
            order = Order.objects.create(user=user, status_id=1)
```
어차피 1번이라면 더 간결하게 지정하셔도 됩니다

Reply:
네 수정했습니다!